classes/mfgtool-initramfs-image.bbclass

```diff
@@ -13,8 +13,8 @@ FEATURE_PACKAGES_extfs = "packagegroup-fsl-mfgtool-extfs"
 FEATURE_PACKAGES_f2fs = "packagegroup-fsl-mfgtool-f2fs"
 
 ZSTD_COMPRESSION_LEVEL ?= "-10"
-IMAGE_FSTYPES = "cpio.zst.u-boot"
-IMAGE_FSTYPES:mxs-generic-bsp = "cpio.gz.u-boot"
+IMAGE_FSTYPES ?= "cpio.zst.u-boot"
```

Comment:
Unfortunately this change makes this line a no-op and breaks the default case.

```
# $IMAGE_FSTYPES [6 operations]
#   set? /.../sources/meta-freescale/conf/machine/include/imx-base.inc:463
#     "${SOC_DEFAULT_IMAGE_FSTYPES}"
#   set /.../sources/poky/meta/conf/documentation.conf:214
#     [doc] "Formats of root filesystem images that you want to have created."
#   set? /.../sources/poky/meta/conf/bitbake.conf:832
#     "tar.gz"
#   set? /.../sources/meta-freescale/classes/mfgtool-initramfs-image.bbclass:16
#     "cpio.zst.u-boot"
#   override[mxs-generic-bsp]:set? /.../sources/meta-freescale/classes/mfgtool-initramfs-image.bbclass:17
#     "cpio.gz.u-boot"
# pre-expansion value:
#     "${SOC_DEFAULT_IMAGE_FSTYPES}"
IMAGE_FSTYPES="wic.bmap  wic.bz2 tar.bz2"
```

Reply:
We can override the SOC_DEFAULT_IMAGE_FSTYPES; it would work.

Reply:
Okay, see https://github.com/Freescale/meta-freescale/pull/1110.